### PR TITLE
First aid

### PR DESCRIPTION
**Pull request by anniviz**
I added a simple contacts page with hard coded contacts (in the next us there will be all the functionality to add/delete/edit own contacts)

### CODE REVIEW

**Review by hugh-burgess — COMMENTED** [review]
LGTM! I couldn’t find any useful tips to give as your method of learning is slightly different to mine, but it is looking very thorough and clean! Well done

File: src/components/ContactGroup.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export default function ContactGroup({ name, content }) {
+  return (
+    <Wrapper>
+      <Title>{name}</Title>
+      <Value>{content}</Value>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.li`
```
Comment (roscha80) [inline src/components/ContactGroup.js:18]: I'm not shure but i think a li needs a ul parent element around.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -45,31 +53,40 @@ const Nav = styled.nav`
...
 
 const MedicationsIconStyled = styled(MedicationsIcon)`
   stroke: ${prop =>
-    prop.$isActive ? 'var(--color-quaternary)' : 'var(--color-primary)'};
-  width: 33px;
-  height: 60px;
+    prop.$isActive ? 'var(--color-gradient-2)' : 'var(--color-primary)'};
+  width: 30px;
+  height: 40px;
+`
```
Comment (roscha80) [inline src/components/Navbar.js:70]: You can put the logic in a var in the globalstyle.css so its a bit more cleaner like this
``` --iconStroke: ${prop => prop.$isActive ? 'var(--color-quaternary)' : 'var(--color-primary)'}```

File: src/pages/ContactPages/ContactsPage.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  )
+}
+
+const Flexbox = styled.ul`
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
+  overflow-y: auto;
+  gap: 20px;
+`
```
Comment (roscha80) [inline src/pages/ContactPages/ContactsPage.js:27]: Same like a bit more up. I guess u need li elements for every listitem

**Review by roscha80 — APPROVED** [review]
Good job Anni :) Only small mentions from me!

File: src/components/ContactGroup.js (hunk 1 above)
Comment (anniviz) [reply src/components/ContactGroup.js:18]: There is a ul as a parent element in the `ContactsPage.js`

File: src/components/Navbar.js (hunk 2 above)
Comment (anniviz) [reply src/components/Navbar.js:70]: I like that!!

File: src/pages/ContactPages/ContactsPage.js (hunk 3 above)
Comment (anniviz) [reply src/pages/ContactPages/ContactsPage.js:27]: They belong together 😉

**Pull request merged**